mon-pix/tests/integration/components/data-protection-policy-information-banner-test.js

```diff
@@ -140,7 +140,7 @@ module('Integration | Component | data-protection-policy-information-banner', fu
               // given
               await setCurrentLocale('nl');
               stubCurrentUserService(this.owner, { shouldSeeDataProtectionPolicyInformationBanner: true });
-              _stubWindowLocationHostname('pix.org');
+              _stubWindowLocationHref('https://pix.org');
```

Comment:
Au lieu de passer par une fonction, on peut directement écrire:

```suggestion
              sinon.stub(Location, 'getHref').returns('https://pix.org');
```